tests/Intl/Normalizer/NormalizerTest.php

```diff
@@ -78,4 +78,66 @@ public function testNormalize()
         $this->assertFalse(pn::normalize($c, -1));
         $this->assertFalse(pn::normalize("\xFF"));
     }
+
+    /**
+     * @covers Symfony\Polyfill\Intl\Normalizer\Normalizer::normalize
+     */
+    public function testNormalizeConformance()
+    {
+        $t = file(__DIR__.'/NormalizationTest.txt');
+        $c = array();
+
+        foreach ($t as $s) {
+            $t = explode('#', $s);
+            $t = explode(';', $t[0]);
+
+            if (6 === count($t)) {
+                foreach ($t as $k => $s) {
+                    $t = explode(' ', $s);
+                    $t = array_map('hexdec', $t);
+                    $t = array_map(__CLASS__.'::chr', $t);
+                    $c[$k] = implode('', $t);
+                }
+
+                $this->assertSame($c[1], pn::normalize($c[0], pn::NFC));
+                $this->assertSame($c[1], pn::normalize($c[1], pn::NFC));
+                $this->assertSame($c[1], pn::normalize($c[2], pn::NFC));
+                $this->assertSame($c[3], pn::normalize($c[3], pn::NFC));
+                $this->assertSame($c[3], pn::normalize($c[4], pn::NFC));
+
+                $this->assertSame($c[2], pn::normalize($c[0], pn::NFD));
+                $this->assertSame($c[2], pn::normalize($c[1], pn::NFD));
+                $this->assertSame($c[2], pn::normalize($c[2], pn::NFD));
+                $this->assertSame($c[4], pn::normalize($c[3], pn::NFD));
+                $this->assertSame($c[4], pn::normalize($c[4], pn::NFD));
+
+                $this->assertSame($c[3], pn::normalize($c[0], pn::NFKC));
+                $this->assertSame($c[3], pn::normalize($c[1], pn::NFKC));
+                $this->assertSame($c[3], pn::normalize($c[2], pn::NFKC));
+                $this->assertSame($c[3], pn::normalize($c[3], pn::NFKC));
+                $this->assertSame($c[3], pn::normalize($c[4], pn::NFKC));
+
+                $this->assertSame($c[4], pn::normalize($c[0], pn::NFKD));
+                $this->assertSame($c[4], pn::normalize($c[1], pn::NFKD));
+                $this->assertSame($c[4], pn::normalize($c[2], pn::NFKD));
+                $this->assertSame($c[4], pn::normalize($c[3], pn::NFKD));
+                $this->assertSame($c[4], pn::normalize($c[4], pn::NFKD));
```

Comment:
should we really call the stub directly here ? Shouldn't it call the polyfilled API instead, relying on the custom test listener ?